pulp_rpm/app/tasks/synchronizing.py

```diff
@@ -110,7 +143,7 @@ class RpmFirstStage(Stage):
     that should exist in the new :class:`~pulpcore.plugin.models.RepositoryVersion`.
     """
 
-    def __init__(self, remote, deferred_download):
+    def __init__(self, remote, deferred_download, new_url=None, kickstart={}):
```

Comment:
Strange bugs can come from a kwarg whose default is a mutable type because if the first time you call it you mutate it, the second time you call it it shows up mutated which is usually unexpected. Generally a `None` as the default is considered a better practice.